src/components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,167 @@
+@import '../Media/Media.scss';
+
+.root {
+  position: relative;
+
+  &.block {
+    margin-left: 5px;
+    display: block;
+  }
+
+  &.inline {
+    display: inline-block;
+    vertical-align: middle;
+  }
+}
+
+/* Tooltip */
+.icon {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.popperContainer {
+  opacity: 0;
+  animation: fadein 0.3s ease-in 0.5s forwards;
+
+  .contentBox {
+    @include for-phone-only {
+      display: none;
+    }
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 12px;
+    text-align: center;
+    width: max-content;
+    max-width: 270px;
+    background-color: white;
+    font-size: 12px;
```

Comment:
There may be a default font def that we can use instead if you don't mind can you look for that and use it instead? Thanks sir!

Reply:
Removed!